File: Functions/gravComp.py
def gravComp(smk, legs, gravVec):
    gravCompTorques = smk.getLegGravCompTorques(legs, gravVec.T)
    gravCompTorques = gravCompTorques[0:3]
    gravCompTorques = gravCompTorques.T
    return gravCompTorques

File: Functions/gravComp.m
function [gravCompTorques] = gravComp(smk, legs, gravVec)
    gravCompTorques = smk.getLegGravCompTorques(legs, gravVec');
    gravCompTorques = gravCompTorques(1:3,:);
    gravCompTorques = (gravCompTorques(:))';
end

